src/CNRichTextEditor.js

```diff
@@ -424,9 +424,10 @@ class CNRichTextEditor extends Component {
             returnKeyType={this.props.returnKeyType}
             foreColor={this.props.foreColor}
             styleList={styles}
-            style={{
+			placeholder={index === 0 && this.props.placeholder}
```

Comment:
I'm using expo on IOS and the package crashes for situations like uploading image. I think its because of this line.